examples/k8s/otel-config.yaml

```diff
@@ -66,7 +66,7 @@ spec:
       - command:
           - "/otelcol"
           - "--config=/conf/otel-agent-config.yaml"
-        image: otel/opentelemetry-collector:0.38.0
+        image: otel/opentelemetry-collector:0.43.0
```

Comment:
Shouldn't it use the latest version available? v0.46.0 at the moment, or v0.47.0 due this week.

Reply:
Should this be added to the checklist for releases?

Reply:
@jpkrohling I have updated the version to 0.47.0 please have a look.